lib/routes.jsx

```diff
@@ -29,6 +29,7 @@ var pages = {
   'clubs/list': require('../pages/clubs-list.jsx'),
   'community': require('../pages/community.jsx'),
   'community/curriculum-workshop': require('../pages/curriculum-workshop.jsx'),
+  'community/curriculum-workshop-march-8-2016': require('../pages/curriculum-workshop-march-8-2016.jsx'),
```

Comment:
double checking - @hannahkane @chadsansing does this url pattern look fine to you? i.e., http://teach.mozilla.org/community/curriculum-workshop-march-8-2016